packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -164,8 +159,8 @@ export const participantListSelector: ParticipantListSelector = createSelector(
       ? convertRemoteParticipantsToParticipantListParticipants(
           updateUserDisplayNamesTrampoline(Object.values(remoteParticipants)),
           localUserCanRemoveOthers,
-          undefined || /* @conditional-compile-remove(hide-attendee-name) */ isHideAttendeeNamesEnabled,
-          undefined || /* @conditional-compile-remove(hide-attendee-name) */ role,
+          undefined || isHideAttendeeNamesEnabled,
```

Comment:
Can remove the `undefined`'s here

Reply:
Ah not my feature so wasn't sure. will look at that